### ISSUE TITLE
EML validation error coming from java exception

### ISSUE THREAD

**Issue by ianbrunjes** [gbif/dwca-validator3#6]
After a few different tests against the GBIF validator, https://tools.gbif.org/dwca-validator/eml.do, it seems like this exception is happening, and being returned as a "Eml validation error". 

```
java.util.MissingResourceException: Could not load any resource bundle by com.sun.org.apache.xerces.internal.impl.msg.XMLSchemaMessages
```

I tested a few xml's, and it happened for each of them- I don't believe the error to be related to the actual content/structure of the EML.

**Comment by mdoering** [comment]
@BrennieDev could you attach a failing EML document please so we can try to reproduce it?

**Comment by ianbrunjes** [comment]
Certainly, though as git doesn't support xml file attachments apparently, I will link to an EML file I tested with here:
https://raw.githubusercontent.com/BrennieDev/HABsDataPublish/master/DwC/datapackage/eml.xml

The error is seen when using the EML Validation tool here: https://tools.gbif.org/dwca-validator/eml.do.

I just tested this validator using a "GBIF annotated metadata" xml downloaded from a GBIF dataset (I used https://www.gbif.org/dataset/1b72b75a-fd98-42de-9d9d-c87c47631c0d), and the error does not occur. So it is possible that the EML I have been trying is somehow invalid- though I have also tested the above file from git in the newer GBIF data validation tool and it claims no issues with the EML.

**Comment by mdoering** [comment]
I first thought it is related to the schema location at https://nis.lternet.edu/schemas/EML/eml-2.2.0/xsd/eml.xsd
But when I remove the location it still shows the error.

> <eml:eml xmlns:eml="https://eml.ecoinformatics.org/eml-2.2.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:stmml="http://www.xml-cml.org/schema/stmml-1.2" xmlns:dc="http://purl.org/dc/terms" xml:lang="eng" packageId="tbd" xsi:schemaLocation="https://eml.ecoinformatics.org/eml-2.2.0  https://nis.lternet.edu/schemas/EML/eml-2.2.0/xsd/eml.xsd" scope="system" system="https://pasta.edirepository.org">
>   <dataset>


One thing to note is that the file is using EML 2.2 which is not supported by GBIF currently.
eml://ecoinformatics.org/eml-2.1.1 is the expected namespace. You can see an example at https://api.gbif.org/v1/dataset/7ddf754f-d193-4cc9-b351-99906754a03b/document
